backend/Squirrel.SqlService/Squirrel.SqlService.BLL/Hubs/SquirrelHub.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Microsoft.AspNetCore.SignalR;
+using Squirrel.Core.BLL.Interfaces;
+using Squirrel.SqlService.BLL.Models.SquirrelHub;
+
+namespace Squirrel.Core.BLL.Hubs;
+
+public sealed class SquirrelHub : Hub<ISquirrelHubToSend>, ISquirrelHubToReceive
+{
+    private ISquirrelHubToReceive _hubToReceiveService;
```

Comment:
What's wrong with namings?) 
And HubNaming)
They explain nothing about what they are doing)

+ using project name in classes makes no sense (maybe except DBContext)

Reply:
Refactored namings and changed implementation of processing received data from console app